src/recipes/vscode/settings.mjs

```diff
@@ -1,17 +1,17 @@
 import { mkdir, readFile, stat, writeFile } from 'fs/promises'
 import { dirname, relative } from 'path'
 
+import { parse, assign, stringify } from 'comment-json'
```

Comment:
This is totally down to personal preference, but I would suggest having a single `JSONC` import here and then use it throughout the file as `JSONC.parse`, `JSONC.stringify`, etc.

Seeing a function called `stringify` or `parse` doesn't give a lot of context and makes the code a little bit harder to parse, IMO.

Reply:
It was a single `CommentJSON` import earlier which is what I prefer. @danez suggested otherwise.

The separate functions seem slightly confusing to me too without any context. WDYT @danez?

Reply:
The problem is that the TS types say that these are named exports. https://github.com/kaelzhang/node-comment-json/blob/master/index.d.ts#L83 So as soon as we switch to TS this will not work, because TS would say `no default export`.

Reply:
Could we do `import * as JSSONC from "json-comments"`?

Reply:
Yeah I guess that should work.